specs/lib/addPipelineExtrasSpec.js

```diff
@@ -8,6 +8,9 @@ describe('addPipelineExtras', function() {
 
     beforeAll(function(done) {
         fs.readFile(gltfExtrasPath, function(err, data) {
+            if (err) {

```

Comment:
There are too many areas that we do this right now, I think I'm just going to save this for the cleanup.